src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -1,26 +1,78 @@
 import * as React from "react"
 import { Box, BoxProps, Flex, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
-import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
 import { FairHeader_fair } from "v2/__generated__/FairHeader_fair.graphql"
+import { ProfileIcon } from "v2/Components/ProfileIcon"
+import styled from "styled-components"
+import { growAndFadeIn, shrinkAndFadeOut } from "v2/Assets/Animations"
+import { Media } from "v2/Utils/Responsive"
+
+const SUBTITLE_HEIGHT = "38px"
 
 interface FairHeaderProps extends BoxProps {
+  stuck?: boolean
   fair: FairHeader_fair
 }
 
-const FairHeader: React.FC<FairHeaderProps> = ({ fair }) => {
-  const { name, exhibitionPeriod } = fair
+const FadingText = styled(Text)<{ show: boolean }>`
+  transform-origin: 0 0;
+  transform: ${p => `scale(${+p.show})`};
+  transition: transform 0.2s;
+  opacity: ${p => +p.show};
+  height: ${p => (p.show ? SUBTITLE_HEIGHT : 0)};
+  animation: ${p =>
+      p.show
+        ? growAndFadeIn(SUBTITLE_HEIGHT)
+        : shrinkAndFadeOut(SUBTITLE_HEIGHT)}
+    0.1s linear;
+`
+
+const ScalingText = styled(Text)<{ stuck: boolean }>`
+  transform-origin: 0 0;
+  transform: ${p => `scale(${p.stuck ? 0.7 : 1})`};
+  transition: transform 0.2s;
+`
 
+const Title: React.FC<{ title: string | null; stuck: boolean }> = ({
+  title,
+  stuck,
+}) => {
   return (
-    <Flex mt={[2, 4]}>
-      <FairHeaderIconFragmentContainer fair={fair} mr={2} />
-      <Box display="flex" flexDirection="column" justifyContent="center">
-        <Text as="h1" variant={["lg", "xl"]}>
-          {name}
+    <>
+      <Media greaterThanOrEqual="md">
+        <ScalingText as="h1" variant={"xl"} stuck={stuck}>
+          {title}
+        </ScalingText>
+      </Media>
+
+      <Media lessThan="md">
+        <Text as="h1" variant="lg">
+          {title}
         </Text>
-        <Text variant={["lg", "xl"]} color="black60">
+      </Media>
+    </>
+  )
+}
+
+const FairHeader: React.FC<FairHeaderProps> = ({ fair, stuck = false }) => {
+  const { name, exhibitionPeriod, profile } = fair
+
+  return (
+    <Flex
+      py={1}
+      border="1px solid transparent"
+      borderBottomColor={stuck ? "black10" : "transparent"}
+    >
```

Comment:
**suggestion**: Might also be worth composing `<Flex>` within Palette's `<BorderBox>` here?